README.md

```diff
@@ -191,35 +191,28 @@ Currently, KADAI provides the following SPIs:
-* [`io.kadai.spi.task.api.AfterRequestReviewProvider`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/AfterRequestReviewProvider.java)
-* [`io.kadai.spi.task.api.BeforeRequestChangesProvider`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/BeforeRequestChangesProvider.java)
-* [`io.kadai.spi.task.api.BeforeRequestReviewProvider`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/BeforeRequestReviewProvider.java)
-* [`io.kadai.spi.task.api.CreateTaskPreprocessor`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/CreateTaskPreprocessor.java)
-* [`io.kadai.spi.task.api.ReviewRequiredProvider`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/ReviewRequiredProvider.java)
-* [`io.kadai.spi.user.api.RefreshUserPostprocessor`](https://github.com/kadai-iokadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/user/api/RefreshUserPostprocessor.java)
+* [`io.kadai.spi.routing.api.TaskRoutingProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/routing/api/TaskRoutingProvider.java)
+* [`io.kadai.spi.task.api.AfterRequestChangesProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/AfterRequestChangesProvider.java)
+* [`io.kadai.spi.task.api.AfterRequestReviewProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/AfterRequestReviewProvider.java)
+* [`io.kadai.spi.task.api.BeforeRequestChangesProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/BeforeRequestChangesProvider.java)
+* [`io.kadai.spi.task.api.BeforeRequestReviewProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/BeforeRequestReviewProvider.java)
+* [`io.kadai.spi.task.api.CreateTaskPreprocessor`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/CreateTaskPreprocessor.java)
+* [`io.kadai.spi.task.api.ReviewRequiredProvider`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/task/api/ReviewRequiredProvider.java)
+* [`io.kadai.spi.user.api.RefreshUserPostprocessor`](https://github.com/kadai-io/kadai/blob/master/lib/kadai-core/src/main/java/io/kadai/spi/user/api/RefreshUserPostprocessor.java)
 
 # 📚Releases
 
-The list of [releases](https://github.com/kadai-iokadai/releases) contains a detailed changelog.
+The list of [releases](https://github.com/kadai-io/kadai/releases) contains a detailed changelog.
 
 We use [Semantic Versioning](https://semver.org/).
 
 The following compatibility matrix shows the officially supported Spring version for each release.
 Other combinations might also work but have not been tested.
```

Comment:
If you delete the compatibility matrix, then you should also probably delete this sentence, I guess

Reply:
Yes you are right